### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by ELAndrews**
#### What does this PR do?
Choose one or more:

- Docs Update

#### Description of Task to be completed? (Copy + Paste Trello card info + link)
Task Description:

README.md file has been reviewed and updated with relevant information. Once the first release canvas has been completed, deployment URLs will need to be updated, as well as initial build, coverage and netlify status URLs.

Trello Card Link: 

none currently available 

#### How has this been tested & how should it be tested in QA?
- The following tests are in place:

Not applicable 

- QA should check:
   
- Reviewed in VSCode preview and checked all links are correct (unless stated above)

#### Any background context you want to provide?

- [X] I have performed a self-review of my own code
- [X] My code follows the style guidelines of this project
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

#### Screenshots (if appropriate)

Screenshot 2020-02-21 at 11 39 05

#### Questions:

- None at the point of PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,27 +94,75 @@ Run `npm test` or `yarn test` in the console.
...
 
 ## Authors
 
-* **Pascal Ulor** - *Team lead* - [github](https://github.com/PascalUlor)
-* **Mariam Adedeji** - *Team member* - [github](https://github.com/mariehposa)
-* **Mike Perry Attara** - *Team member* - [github](https://github.com/mikeattara)
-* **Justinas Vebra** - *Team member* - [github](https://github.com/vebradev)
-* **Mildred Otieno** - *Team member* - [github](https://github.com/awuorm)
-* **Ayomide Jones** - *Team member* - [github](https://github.com/Ayormeday)
+| [Anthony Campbell](https://anthonycampbell.dev) | [AbdelIdir ](https://github.com/AbdelIdir)      | [Ekanem David](https://github.com/dueka)        |
```
Comment (AnthonyJCampbell) [inline README.md]: Could you please add `Team Lead - ......` in front of my name to denote clear role distinction?

File: README.md [hunk 2]
```diff
@@ -91,27 +94,75 @@ Run `npm test` or `yarn test` in the console.
...
+|                                                                          [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/emma-andrews-b26768169/)                                                                           |                           [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/john-afolabi/)                           |       [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/karim-bertacche-64b270156/)       |
 
 ## Acknowledgments
 
 * Lambda
```
Comment (AnthonyJCampbell) [inline README.md]: Perhaps we can add a link to Lambda School (https://lambdaschool.com) & a link to Alex (in his position as Engineering Manager)

File: README.md [hunk 3]
```diff
@@ -91,27 +94,75 @@ Run `npm test` or `yarn test` in the console.
 * [Query-string](https://www.npmjs.com/package/query-string) - Used to retrieve the variable values in the HTTP query string.
 * [Eslint](https://eslint.org/) - A tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
 
+## Versioning
+
+This is version 1.0 (Release canvas 1)
```
Comment (AnthonyJCampbell) [inline README.md]: I would personally not explicitly mention the release canvas but to merely have it be evident in the version number. I would opt for `V 2.0.1` (reflecting that we're using someone's existing codebase)

File: README.md [hunk 4]
```diff
@@ -50,10 +47,26 @@ Version Control (Git)
...
+* Next install the package dependencies by running; this can be `npm` or `yarn
+
+```bash
+npm install
+```
+
+* After that start or build the application by running `npm start` or `yarn start`
+
+```bash
+yarn start # For a local development
```
Comment (AnthonyJCampbell) [inline README.md]: I would make this `npm start` in order to have consistency with the other two command above

File: README.md (hunk 1 above)
Comment (ELAndrews) [reply README.md]: After discussing with others within the group, this has been implemented by adding an additional row stating all posts. This has now been completed.

File: README.md (hunk 2 above)
Comment (ELAndrews) [reply README.md]: Completed

File: README.md (hunk 3 above)
Comment (ELAndrews) [reply README.md]: Completed

File: README.md (hunk 4 above)
Comment (ELAndrews) [reply README.md]: Completed

File: README.md (hunk 1 above)
Comment (ELAndrews) [reply README.md]: After a discussion with the group, we have added in another row with identifies everyone's role. 
Commpleted

**Pull request merged**